conans/model/conan_file.py

```diff
@@ -209,8 +210,19 @@ def buildenv(self):
             self._conan_buildenv = self._conan_buildenv.get_profile_env(ref_str)
         return self._conan_buildenv
 
-    def initialize(self, settings, env, buildenv=None):
+    @property
+    def runenv(self):
+        # Lazy computation of the package buildenv based on the profileone
```

Comment:
The comment mentions `buildenv`, should it be `runenv`?

Reply:
```suggestion
        # Lazy computation of the package runenv based on the profile one
```
?

Reply:
yes, comment was not updated